packages/schematics/angular/application/files/tsconfig.spec.json

```diff
@@ -1,7 +1,7 @@
 {
-  "extends": "<%= sourcedir.split('/').map(x => '..').join('/') %>/tsconfig.json",
+  "extends": "<%= newProjectRoot.split('/').map(x => '..').join('/') %>/tsconfig.json",
```

Comment:
shouldn't that include `sourcedir`?

Reply:
I added a couple of tests, it should be good.